harvard-lts/etd_dash_service
# Etd 266

**Use os-level unzip/zip.**
* * *

**JIRA Ticket**: [(link)](https://at-harvard.atlassian.net/browse/ETD-266)

# What does this Pull Request do?
This pr reverts to using unix os level zip and unzip (python zip was garbling file titles in some cases). Has unit tests to unzip, test the title; then zip, because thats the next step in the process; finally unzip and test the title again to confirm zip also hasn't corrupted.

# How should this be tested?
Check out this branch
Copy .env.example to .env and fill for dev
Start container: docker-compose -f docker-compose-local.yml up --build -d --force-recreate
Exec into container: docker exec -it etd-dash-service bash
Run > pytest
Tests should pass

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Y
- integration tests? Manual testing, see jira

# Interested parties
Tag (@ mention) interested parties
